### PR TITLE
soccer game added

### PR DESCRIPTION
**Pull request by bhumika-1127**
@kunjgit I have added a soccer game wrt issue #4198.Kindly review the PR.

Screenshot 2024-06-06 154128

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @bhumika-1127 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by bhumika-1127** [comment]
\assign

**Comment by bhumika-1127** [comment]
@kunjgit Kindly review the pr.

File: assets/js/gamesData.json [hunk 1]
```diff
@@ -2102,11 +2102,22 @@
  "gameTitle": "Guess_The_Song",
   "gameUrl": "Guess_The_Song",
   "thumbnailUrl": "Guess_The_Song.png"
+  },"421":{"gameTitle": "Pen_Pointer_Fight",
```
Comment (ishita-43) [inline assets/js/gamesData.json:2105]: remove this one if not intended!!

Comment (ishita-43) [reply assets/js/gamesData.json:2105]: keep your project entry only

File: assets/js/gamesData.json [hunk 2]
```diff
@@ -2102,11 +2102,22 @@
...
+  "gameUrl": "PenPointerFight",
+  "thumbnailUrl": "PenPointerFight.png"
+},"422":{
+  "gameTitle": "MathQuiz",
+  "gameUrl": "MathQuiz",
+  "thumbnailUrl": "MathQuiz.png"
+},"423":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2112]: after removing other your entry should be below 417 as 423

File: assets/js/index.js
Comment (ishita-43) [inline assets/js/index.js:1]: if these changes are not compulsory dont make it

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make changes soon!!

File: assets/js/index.js
Comment (bhumika-1127) [reply assets/js/index.js:1]: @ishita-43 I don't get it I have added my game in it, if I remove other then those game might not be visible. Please elaborate what is to be done.

Comment (ishita-43) [reply assets/js/index.js:1]: there is no 418 in the entry the last entry we saw is 417 in the above file..

File: assets/js/gamesData.json (hunk 2 above)
Comment (ishita-43) [reply assets/js/gamesData.json:2112]: did you added 421,422 also by yourself!!

File: assets/js/index.js
Comment (bhumika-1127) [reply assets/js/index.js:1]: so next in sequence should be 418?

Comment (ishita-43) [reply assets/js/index.js:1]: it is not incresing like that 
if you look carefully the pattern is like this
406
415
407
416
408
417
409
418
and so on 
so if you are willing to add your game before 417 name it as 408
and if you yourself added morethan one game remove them from there or place them in similar pattern

File: assets/js/gamesData.json [hunk 3]
```diff
@@ -2102,22 +2102,22 @@
...
-},"422":{
-  "gameTitle": "MathQuiz",
-  "gameUrl": "MathQuiz",
-  "thumbnailUrl": "MathQuiz.png"
-},"423":{
-   "gameTitle": "Soccer",
-   "gameUrl": "Soccer",
-   "thumbnailUrl": "Soccer"
-}
+  }
```
Comment (ishita-43) [inline assets/js/gamesData.json:2105]: 408 here one game

File: assets/js/gamesData.json [hunk 4]
```diff
@@ -2102,22 +2102,22 @@
...
 },"417":{
   "gameTitle": "Brick Buster",
    "gameUrl": "Brick Buster",
    "thumbnailUrl": "Brick.png"
- }
+ },"418":{
```
Comment (ishita-43) [inline assets/js/gamesData.json]: 409 here second entry

File: assets/js/gamesData.json [hunk 5]
```diff
@@ -2102,22 +2102,22 @@
...
+  "gameTitle": "Soccer",
+  "gameUrl": "Soccer",
+  "thumbnailUrl": "Soccer"
+},"419":{"gameTitle": "Pen_Pointer_Fight",
```
Comment (ishita-43) [inline assets/js/gamesData.json]: should be as 418

File: assets/js/gamesData.json [hunk 6]
```diff
@@ -2102,22 +2102,22 @@
...
+"gameUrl": "PenPointerFight",
+"thumbnailUrl": "PenPointerFight.png"
+},"420":{
```
Comment (ishita-43) [inline assets/js/gamesData.json]: place this one in above space for 408 and remove from here also change label to 408

File: assets/js/index.js
Comment (bhumika-1127) [reply assets/js/index.js:1]: @ishita-43 I have made the changes, kindly let me know it anything else is to be done.

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @bhumika-1127 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**